### PR TITLE
[MRESOLVER-281] Update configuration page with newly introduced elements

### PR DESCRIPTION
**Pull request by cstamas**
In 1.9.0 we got several new components that has configuration keys, document them.

### CODE REVIEW

File: src/site/markdown/configuration.md [hunk 1]
```diff
@@ -21,6 +21,12 @@ under the License.
 Option | Type | Description | Default Value | Supports Repo ID Suffix
 --- | --- | --- | --- | ---
 `aether.artifactResolver.snapshotNormalization` | boolean | It replaces the timestamped snapshot file name with a filename containing the `SNAPSHOT` qualifier only. This only affects resolving/retrieving artifacts but not uploading those. | `true` | no
+`aether.artifactResolver.simpleLrmInterop` | boolean | Enable interop with Simple LRM. Ignored when RRF used. | `false` | no
+`aether.artifactResolver.postProcessor.trusted-checksums` | boolean | Enable `trusted-checksums` resolver post processor. | `false` | no 
+`aether.artifactResolver.postProcessor.trusted-checksums.checksumAlgorithms` | String | Comma separated list of checksum algorithms with which `trusted-checksums` should operate (validate or record). | `"SHA-1"` | no
```
Comment (michael-o) [inline src/site/markdown/configuration.md:26]: Comma-separated

Seems like this is inconsistent throughout

File: src/site/markdown/configuration.md [hunk 2]
```diff
@@ -21,6 +21,12 @@ under the License.
 Option | Type | Description | Default Value | Supports Repo ID Suffix
 --- | --- | --- | --- | ---
 `aether.artifactResolver.snapshotNormalization` | boolean | It replaces the timestamped snapshot file name with a filename containing the `SNAPSHOT` qualifier only. This only affects resolving/retrieving artifacts but not uploading those. | `true` | no
+`aether.artifactResolver.simpleLrmInterop` | boolean | Enable interop with Simple LRM. Ignored when RRF used. | `false` | no
+`aether.artifactResolver.postProcessor.trusted-checksums` | boolean | Enable `trusted-checksums` resolver post processor. | `false` | no 
+`aether.artifactResolver.postProcessor.trusted-checksums.checksumAlgorithms` | String | Comma separated list of checksum algorithms with which `trusted-checksums` should operate (validate or record). | `"SHA-1"` | no 
+`aether.artifactResolver.postProcessor.trusted-checksums.failIfMissing` | boolean | Makes `trusted-checksums` fail validation if a trusted checksum for artifact is missing. | `false` | no
```
Comment (michael-o) [inline src/site/markdown/configuration.md:27]: for an artifact

File: src/site/markdown/configuration.md (hunk 1 above)
Comment (cstamas) [reply src/site/markdown/configuration.md]: fixed and made all comma-separated

File: src/site/markdown/configuration.md (hunk 2 above)
Comment (cstamas) [reply src/site/markdown/configuration.md]: fixed

**Review by michael-o — APPROVED** [review]

**Comment by jira-importer** [comment]
Resolve #956

**Comment by jira-importer** [comment]
Resolve #956

**Pull request merged**